### PR TITLE
Initial user journey test for bookshelf-flex

### PR DESCRIPTION
**Pull request by jerjou**

### CODE REVIEW

File: bookshelf-standard/2-structured-data/pom.xml [hunk 1]
```diff
@@ -134,10 +134,6 @@ Copyright 2016 Google Inc.
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-server</artifactId>
     </dependency>
-    <dependency>
```
Comment (lesv) [inline bookshelf-standard/2-structured-data/pom.xml]: This is removing this from bookshelf standard -- is that what you want?

**Comment by lesv** [comment]
Please fix Circle before merging.

File: bookshelf-standard/2-structured-data/pom.xml (hunk 1 above)
Comment (jerjou) [inline bookshelf-standard/2-structured-data/pom.xml]: Yeah - I realized that the dependency wasn't actually necessary :-) Apparently it's only necessary if you want to do chrome- or firefox- specific stuff, but I'm only using the generic RemoteWebDriver (which for all I know includes this already as a dependency. Anyway, it works without this dep, so I figured I'd clean up)

**Comment by jerjou** [comment]
Okay - ended up having to add tests for all the flex bookshelf steps, I suspect because spinning up jetty and immediately shutting it down sometimes doesn't work. The tests are just skeletons for now, and I'll fill them out in subsequent PRs.

Also ended up adding `selenium-chrome-driver` back in - turns out we do need it? Or at least, without it I was getting intermittent errors where some HTTP lib wasn't being found. I figured I could include that dependency (which is kind of random), or I could include the chromedriver one, which makes more sense and also resolves that issue.

**Pull request merged**